views/applicationForms/applicationType.ejs

```diff
@@ -1,76 +1,181 @@
+                        </p>
+                        <ul class="govuk-list govuk-list--bullet govuk-list--spaced govuk-!-margin-bottom-0">
+                            <li>upload documents you want to legalise, for example a passport, as a PDF</li>
+                            <li>upload multiple documents in a single PDF (for example a passport and a driving licence)</li>
+                            <li>pay £30 for each uploaded PDF, even if it contains multiple documents</li>
+                            <li>upload up to 50 PDFs in a single application</li>
+                            <li>avoid courier or postage costs</li>
+                        </ul>
+                        <p>
+                            You need to:
+                        </p>
+                        <ul class="govuk-list govuk-list--bullet govuk-list--spaced">
+                            <li>check if an e-Apostille will be accepted in the country in which you want to use it (no refund if an e-Apostille is not
+                            accepted)</li>
+                            <li>ensure all PDFs are notarised and digitally signed by a notary</li>
+                            <li>download the PDFs (including e-Apostilles) from your account within 21 days of the documents being legalised</li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="govuk-radios__item">
+                    <input class="govuk-radios__input" id="choose-standard-service" name="choose-a-service" type="radio" value="standard">
+                    <label class="govuk-label govuk-radios__label" for="choose-standard-service">
+                        <h2 class="heading-medium govuk-!-margin-top-0">Standard service (paper based)</h2>
+                    </label>
+                    <div class="govuk-!-padding-left-3">
+                        <p>
+                            The standard service involves applying online and posting the documents you wish to legalise.
+                        </p>
+                        <ul class="govuk-list govuk-list--bullet govuk-list--spaced">
+                            <li>you need to post the documents you wish to legalise to the Legalisation Office</li>
+                            <li>your legalised documents and Apostille will be returned to your chosen address by courier or post</li>
+                            <li>it costs £30 per document, plus courier fees or postage</li>
+                        </ul>
+                    </div>
 
-</div>
+                    <details class="govuk-details" data-module="govuk-details">
+                        <summary class="govuk-details__summary">
+                            <span class="govuk-details__summary-text">
+                                How much are the courier fees or postage?
+                            </span>
+                        </summary>
+                        <div class="govuk-details__text">
```

Comment:
Not sure how much this information changes but may want to consider splitting the courier variables into config so you can change them in the config and they get applied in whatever pages need them.

Reply:
According to Kainos, these values change so rarely it won't make that much of a different to put them in the config. 
However, it might make more sense to read the prices from the `PostagesAvailable ` table in the DB.

As I don't have any experience with that table, are you happy for me to create a different ticket for this specific change?

Reply:
No problem.  It was just a thought so it sounds like there's actually not much benefit in making the change - your call to raise a separate ticket or scrap it.